cves/CVE-2017-5648.yml

```diff
@@ -64,9 +72,9 @@ fixes:
    - commit: 0f7b9465d594b9814e1853d1e3a6e3aa51a21610
      note: SVN rev 1785775, from the Tomcat website.
    - commit: 6bb36dfdf6444efda074893dff493b9eb3648808
-     note: SVN rev 1785777, from the Tomcat website.
-   - commit:
-     note:
+     note: SVN rev 1785775, from the Tomcat website.
+   - commit: c38b7baaa710a7c876aef47a27b97614ad9efe60
+     note: SVN rev 391432, from the Tomcat website.
 vccs:
```

Comment:
Please fill in the VCC. This is an extremely important part of the assignment.

Reply:
added to fixed